Fst2.h

```diff
@@ -157,6 +157,13 @@ struct fst2Tag {
      * tokens, and compound words will be handled via a compound pattern.
      */
     int compound_pattern;
+
+    /*
+     * This field is used to interrupt the exploration of the path
+     * when a lexical mask does not find any entry in the morphological dictionaries
+     */
+    bool stop;
+
```

Comment:
Avoid vertical space at the end of a struct declaration

Reply:
done